product_docs/docs/pgd/5/durability/commit-scopes.mdx

```diff
@@ -144,51 +144,27 @@ commit_scope_kind:
     * `max_commit_delay` (interval) &mdash; Maximum delay that can be injected to commit to try to keep within the lag limits.
 
 !!! Note
-    `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause as well, making `CAMO` syntax deprecated.
+The `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with an additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause, making `CAMO` syntax deprecated.
```

Comment:
I'm unsure if "syntactic sugar" is an appropriate description here. How about "more readable/convenient" or "easier to understand"?

Reply:
Moved the statement anyway to the CAMO section, removed the future promise and changed to "is mostly an alias for"